src/components/ebay-menu/index.js

```diff
@@ -17,19 +17,25 @@ function getInitialState(input) {
     const type = input.type;
     const isRadio = type === 'radio';
     const isCheckbox = type === 'checkbox';
+    const isFake = type === 'fake';
     let checkedItems = [];
 
     const items = (input.items || []).map((item, i) => {
         const classes = [item.class];
         const href = item.href;
+        const itemType = item.type; // for button menu item in fake menu
         const checked = item.checked;
         const current = item.current; // only for fake menu items
         let role;
         let tag;
 
-        if (href) {
-            tag = 'a';
+        if (isFake) {
             classes.push('fake-menu__item');
+            if (href) {
```

Comment:
Should this be two types, `button` and `link`? Otherwise the developer has to think hard about how to create the right kind of menu item they want.

Reply:
I tend to agree here.

Reply:
I'm ok with allowing `link`, it's just that it doesn't seem like it would actually do anything. If you have a link menu-item, you have to send an `href` no matter what (right?). If we allow `type=link`, would that mean that you have to send both, or else we don't treat it like a link? Or do we still treat it as link even if you only specify `href`?

Reply:
To me, it doesn't make sense to have `type` for only one thing. If we have a type, we should allow the developer to specify it for all types. Otherwise, _their logic_ gets really cumbersome.

Reply:
I agree with Sean. I even think we should allow type="menuitem" on the root. Even though menu item is the default (not checkbox or radio). This is like HTML, the default can be specified or not.

Reply:
This can be taken on as a separate PR, if necessary, to achieve consistency across all components.